### ISSUE TITLE
Animation issue using when Ionic 3 with '--prod' flag

### ISSUE THREAD

**Issue by sharecube** [Shailu4u/ng-masonry-grid#10]
The problem is when we using module in Ionic 3 with '--prod' flag there is animation issue when removing items: animation calls twice. Without '--prod' mode it works well.

**Comment by Shailu4u** [comment]
@sharecube , can you specify if any errors using --prod flag.

**Comment by sharecube** [comment]
No, it builds for android without any errors. There is an 'ionic info' and link to video from device (Android 5.1) with this issue:

https://www.youtube.com/watch?v=XqA5ir14uMY&feature=youtu.be

I

    @ionic/cli-utils  : 1.19.2
    ionic (Ionic CLI) : 3.20.0

global packages:

    cordova (Cordova CLI) : 8.0.0

local packages:

    @ionic/app-scripts : 3.1.8
    Cordova Platforms  : android 7.0.0
    Ionic Framework    : ionic-angular 3.9.2

System:

    Node : v8.10.0
    npm  : 5.6.0
    OS   : Windows 10

Environment Variables:

    ANDROID_HOME : not set

Misc:

    backend : pro

**Comment by Shailu4u** [comment]
@sharecube did you tried in real device or emulator?. Can you give if any console errors on remove item.
I had seen your video, I think transitionend event is not firing properly.

**Comment by sandiphob** [comment]
A work-around I did was to turn off the animation by setting transitionDuration: 0 in masonryOptions. And then applying a css animation class directly ng-masonry-grid-item

**Comment by wynfred** [comment]
maybe related to https://github.com/gethinoakes/ngx-masonry/issues/8
